examples/ts/nextjs/server.js

```diff
@@ -0,0 +1,28 @@
+require('./lib/bugsnag');
+const Bugsnag = require ('@bugsnag/js');
+const next = require('next')
+const express = require('express')
+
+const port = parseInt(process.env.PORT, 10) || 3000
+const dev = process.env.NODE_ENV !== 'production'
+const app = next({ dev })
+const handler = app.getRequestHandler()
+
+var middleware = Bugsnag.getPlugin('express')
+
+app.prepare().then(() => {
+
+  express()
+    .use(middleware.requestHandler)
+    .get('/custom-server/bad', (req, res) => {
+      throw new Error('exception in custom express server (outside next.js)')
+    })
+    .use(middleware.errorHandler)
```

Comment:
Handle errors in the custom express server before we hit the next.js handler (as that handles it's own errors).

We might want to move this out the example to simplify things a bit and focus on the more common next.js use cases i.e. static and serverless.